### PR TITLE
Update corefx stages status

### PR DESCRIPTION
**Pull request by safern**
cc: @danmosemsft

### CODE REVIEW

**Comment by riarenas** [comment]
Thanks! Let us know if you hit any blockers along the way

File: Documentation/YamlStagesRepoStatus.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Target completion date is 8/13/2019.
 | CLI                        | licavalc         | At risk | ❌ |  No plan available |
 | CLICommandLineParser       | licavalc         | At risk | ❌ |  No plan available |
 | CoreClr                    | russellk         | At risk | ❌ |  No plan available |
-| CoreFx                     | danmose/safern   | At risk | ❌ |  No plan available |
+| CoreFx                     | danmose/safern   | On track | ➖ | |
```
Comment (riarenas) [inline Documentation/YamlStagesRepoStatus.md]: Do you have a sense for when you think it will be done?

Comment (safern) [reply Documentation/YamlStagesRepoStatus.md]: End of next week the latest. (Being conservative, should be earlier).

Comment (riarenas) [inline Documentation/YamlStagesRepoStatus.md]:
```suggestion
| CoreFx                     | danmose/safern   | On track | ➖ | Expected completion: 8/9 |
```

**Pull request merged**